mlflow/mleap.py

```diff
@@ -12,13 +12,24 @@
 import os
```

Comment:
@harupy MLeap models can't currently be loaded in Python. I don't think we need the `conda_env`, etc arguments here.

Reply:
Got it, I'll revert the changes on `mleap.py`.

Reply:
Done!